lib/shift_generator.rb

```diff
@@ -0,0 +1,25 @@
+require 'date'
```

Comment:
now able to pull all dependency requires from enigma file over to shift_generator as this module utilizes each one making it available to the main class